pkg/relay/ethereum/module.go

```diff
@@ -24,3 +27,8 @@ func (Module) RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 func (Module) GetCmd(ctx *config.Context) *cobra.Command {
 	return nil
 }
+
+func GetModuleLogger() *log.RelayLogger {
+	return log.GetLogger().
+		WithModule(LOGGER_MODULE)
+}
```

Comment:
You are using this module logger everywhere in this module, but this logger contains no information about clients or connecitons or channels or anything else...?
Readers of logs can't even know which chain (src or dst) is processed, can they?

Basically we want to use `core.GetXxxLogger` also here.
If `core.GetXxxLogger` is not suitable for modules outside of core, please fix them.

Reply:
Add `WithEthereumChain` to yui-relayer logger
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/06014ce49818bb0e597d3e6c6e8c22d89b0923f9